### PR TITLE
[WIP] git contributors email and organization maps

### PR DESCRIPTION
**Pull request by mrpollo**
The following files can be used to provide an accurate contributors report as well as a report that can show the number of commits per registered company.

- [ ] Validate the mail map data with developers
- [ ] Validate the organizational data with developers
- [ ] provide a way to track developer contributions attributing the right company over time

Contributor list (clean) 1 year
```
git log --pretty='%aN <%aE>' --since="1 year" | sort | uniq -c | sort -rn

 858 Daniel Agar <daniel@agar.ca>
 473 Beat Küng <beat-kueng@gmx.net>
 345 Dennis Mannhart <dennis@yuneecresearch.com>
 264 David Sidrane <david_s5@nscdg.com>
 178 Matthias Grob <maetugr@gmail.com>
 151 Lorenz Meier <lorenz@px4.io>
 112 Nuno Marques <n.marques21@hotmail.com>
 109 Roman Bapst <roman@px4.io>
  96 Paul Riseborough <p_riseborough@live.com.au>
  92 Martina Rivizzigno <martina@rivizzigno.it>
  70 Florian Achermann <florian.achermann@mavt.ethz.ch>
  70 Christoph Tobler <christoph@yuneecresearch.com>
  66 PX4 Bots <bot@px4.io>
  61 Daniele Pettenuzzo <daniele@px4.io>
  57 Alessandro Simovic <alessandro@yuneecresearch.com>
  50 Mark Sauder <mcsauder@gmail.com>
  45 Simone Guscetti <simone@yuneecresearch.com>
  42 Julian Oes <julian@oes.ch>
  39 Julien Lecoeur <julien.lecoeur@gmail.com>
  35 Anthony Lamping <lamping.ap@gmail.com>
  30 Mathieu Bresciani <brescianimathieu@gmail.com>
  29 Jake Dahl <jacob.dahl@tealdrones.com>
  28 Philipp Oettershagen <philipp.oettershagen@mavt.ethz.ch>
  28 Hamish Willee <hamishwillee@gmail.com>
  25 Mohammed Kabir <kabir@uasys.io>
  21 Nicolas de Palezieux <ndepal@gmail.com>
  14 Sander Smeets <sander@droneslab.com>
  13 Johannes Brand <johannes@auterion.com>
  11 Bob-F <38539206+UAV-Pilot@users.noreply.github.com>
   9 Oleg Kalachev <okalachev@gmail.com>
   9 Carl Olsson <carlolsson.co@gmail.com>
   9 Andreas Antener <antener_a@gmx.ch>
   8 SungTae Moon <munhoney@gmail.com>
   7 Barza Nisar <nisarb@student.ethz.ch>
   7 Bart Slinger <bartslinger@gmail.com>
   6 rolandash <ning.roland@gmail.com>
   6 Thomas Stastny <thomas.stastny@mavt.ethz.ch>
   6 Amir Melzer <amir.melzer@mavt.ethz.ch>
   5 Kurt Kiefer <kekiefer@gmail.com>
   5 Bram Strack van Schijndel <bramsvs@gmail.com>
   4 Simon Laube <simon@leitwert.ch>
   4 Ramón Roche <mrpollo@gmail.com>
   4 Nick Anthony <nickmanthony@hotmail.com>
   4 Matthew Edwards <mje-nz@users.noreply.github.com>
   4 Karl Schwabe <ksschwabe@gmail.com>
   4 Elia Tarasov <elias.tarasov@gmail.com>
   4 David Royer <dave@aerotenna.com>
   4 Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>
   3 makekam <1256400447@qq.com>
   3 dakejahl <37091262+dakejahl@users.noreply.github.com>
   3 Phillip Kocmoud <phil@itchy.io>
   3 Nathan Tsoi <nathan@vertile.com>
   3 Lukas Woodtli <woodtli.lukas@gmail.com>
   3 José Roberto de Souza <jose.souza@intel.com>
   3 Friedrich Beckmann <friedrich.beckmann@gmx.de>
   3 Alex Klimaj <alex.klimaj@tealdrones.com>
   2 yinjiajie <yinjiajie_adi@163.com>
   2 korigod <akorigod@gmail.com>
   2 garfieldG <garfieldG@users.noreply.github.com>
   2 Zack Selgrath <zselgrath@gmail.com>
   2 Vasily Evseenko <svpcom@gmail.com>
   2 Ritul Jasuja <ritul.jasuja@intel.com>
   2 Pietro De Nicolao <pietrodn@users.noreply.github.com>
   2 Matthew Dailey <dailey.matthew@gmail.com>
   2 Jan Okle <jan@leitwert.ch>
   2 Ivo Drescher <ivo.drescher@gmail.com>
   2 Dion Gonano <diongonano@gmail.com>
   2 David Riseborough <drisebor@hotmail.com>
   2 Ayush <ayush.gaud@gmail.com>
   1 vkotaru <prasanth.kotaru@gmail.com>
   1 mirkix <mirkix@gmail.com>
   1 githubDLG <oneshot.triplekill@qq.com>
   1 dw.xiong <1308455330@qq.com>
   1 Vicente Monge <vicentemonge@eprosima.com>
   1 Trevor Joynson <github@trevor.joynson.io>
   1 ToppingXu <toppingxu@outlook.com>
   1 Sugnan Prabhu <sugnan.prabhu.s@intel.com>
   1 Shivam Chauhan <iamchauhan55@gmail.com>
   1 Sebastian Verling <sverling@student.ethz.ch>
   1 Robbie Sharma <robbie.s@inanibus.com>
   1 Ricardo Marques <marques.ricardo17@gmail.com>
   1 Nick Anthony <nickexists@hotmail.com>
   1 Mateusz Sadowski <msadowski90@gmail.com>
   1 Martin Trgina <martin.trgina@belladati.com>
   1 Kunal Shah <kshah.kunal@gmail.com>
   1 Kevin Lopez Alvarez <kevin.lopez.alvarez@drotek.com>
   1 Keenan Johnson <keenan.johnson@gmail.com>
   1 Jun-Tao <jt.li@yuneec.com>
   1 Joshua Whitehead <josh.whitehead@dornerworks.com>
   1 Jonas Vautherin <jonas.vautherin@gmail.com>
   1 James Goppert <james.goppert@gmail.com>
   1 Huang <huangwenfuture@sina.com>
   1 Haukanes <aksel.haukanes@gmail.com>
   1 Giovanni Carbone <gianni.carbone@gmail.com>
   1 Don Gagne <don@thegagnes.com>
   1 Coby <43158+cglusky@users.noreply.github.com>
   1 Anass Al <mr@anassinator.com>
   1 Alexis Paques <alexis.paques@gmail.com>
   1 Alexandr Kondratev <theg4sh@gmail.com>
```

Organization list 1 year
```
git log --pretty='%aN <%aE>' --since="1 year" | git -c mailmap.file=.organizationmap check-mailmap --stdin | sort | uniq -c | sort -rn

1307 Unaffiliated <no@organization.net>
 969 Auterion <contact@auterion.com>
 829 Yuneec Research <contact@yuneecresearch.com>
 106 ETH Zurich <contact@student.ethz.ch>
  82 Teal Drones <contact@tealdrones.com>
  66 PX4 Bots <bot@px4.io>
  39 EPFL <contact@epfl.ch>
  25 MIT <contact@mit.edu>
  21 Nicolas de Palezieux <ndepal@gmail.com>
  14 Vertical Technologies <contact@verticaltechnologies.com>
  13 Copter Express <contact@copterexpress.com>
  11 Bob-F <38539206+UAV-Pilot@users.noreply.github.com>
  10 Wingtra <contact@wingtra.com>
  10 Intel <contact@intel.com>
   9 UAVenture <contact@uaventure.com>
   7 Bart Slinger <bartslinger@gmail.com>
   6 Leitwert <contact@leitwert.ch>
   5 Kurt Kiefer <kekiefer@gmail.com>
   5 Bram Strack van Schijndel <bramsvs@gmail.com>
   4 Matthew Edwards <mje-nz@users.noreply.github.com>
   4 Fotokite <contact@fotokite.com>
   4 Dronecode <contact@dronecode.com>
   4 Aerotenna <contact@aerotenna>
   3 makekam <1256400447@qq.com>
   3 dakejahl <37091262+dakejahl@users.noreply.github.com>
   3 Lukas Woodtli <woodtli.lukas@gmail.com>
   3 Friedrich Beckmann <friedrich.beckmann@gmx.de>
   2 yinjiajie <yinjiajie_adi@163.com>
   2 korigod <akorigod@gmail.com>
   2 garfieldG <garfieldG@users.noreply.github.com>
   2 Zack Selgrath <zselgrath@gmail.com>
   2 Vasily Evseenko <svpcom@gmail.com>
   2 Pietro De Nicolao <pietrodn@users.noreply.github.com>
   2 Matthew Dailey <dailey.matthew@gmail.com>
   2 Dion Gonano <diongonano@gmail.com>
   2 Ayush <ayush.gaud@gmail.com>
   1 vkotaru <prasanth.kotaru@gmail.com>
   1 mirkix <mirkix@gmail.com>
   1 Yuneec <contact@yuneec.com>
   1 Vicente Monge <vicentemonge@eprosima.com>
   1 Trevor Joynson <github@trevor.joynson.io>
   1 ToppingXu <toppingxu@outlook.com>
   1 Shivam Chauhan <iamchauhan55@gmail.com>
   1 Robbie Sharma <robbie.s@inanibus.com>
   1 Ricardo Marques <marques.ricardo17@gmail.com>
   1 Nick Anthony <nickexists@hotmail.com>
   1 Kunal Shah <kshah.kunal@gmail.com>
   1 James Goppert <james.goppert@gmail.com>
   1 Impossible Aerospace <contact@impossible.aero>
   1 Huang <huangwenfuture@sina.com>
   1 Haukanes <aksel.haukanes@gmail.com>
   1 Giovanni Carbone <gianni.carbone@gmail.com>
   1 Drotek <contact@drotek.com>
   1 Coby <43158+cglusky@users.noreply.github.com>
   1 BellaDati <contact@belladati.com>
   1 Anass Al <mr@anassinator.com>
   1 Alexis Paques <alexis.paques@gmail.com>
```

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# PX4 .mailmap
+# See this guide for help modifying this file https://github.com/sympy/sympy/wiki/Using-.mailmap
+# Please append your entry
+# Example use of this map:
+#   git log --pretty='%aN <%aE>' | sort | uniq -c | sort -rn | HEAD -10
+
+Alessandro Simovic <alessandro@yuneecresearch.com> alessandro <alessandro@yuneecresearch.com>
+Alessandro Simovic <alessandro@yuneecresearch.com> alessandro <3762382+potaito@users.noreply.github.com>
+Alexandr Kondratev <theg4sh@gmail.com> <theg4sh@gmail.com>
+Amir Melzer <amir.melzer@mavt.ethz.ch> <amir.melzer@mavt.ethz.ch>
```
Comment (LorenzMeier) [inline .mailmap]: ETH Zurich

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Anthony Lamping <lamping.ap@gmail.com> <lamping.ap@gmail.com>
+Beat Küng <beat-kueng@gmx.net> <beat-kueng@gmx.net>
```
Comment (LorenzMeier) [inline .mailmap]: Auterion

File: .mailmap [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Carl Olsson <carlolsson.co@gmail.com> <carlolsson.co@gmail.com>
```
Comment (LorenzMeier) [inline .mailmap]: Wingtra

File: .mailmap [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Daniel Agar <daniel@agar.ca> <daniel@agar.ca>
+Daniele Pettenuzzo <daniele@px4.io> Daniele Pettenuzzo <dani.pettenuzzo@gmail.com>
```
Comment (LorenzMeier) [inline .mailmap]: Auterion

File: .mailmap [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Daniele Pettenuzzo <daniele@px4.io> Daniele Pettenuzzo <35664507+DanielePettenuzzo@users.noreply.github.com>
+Daniele Pettenuzzo <daniele@px4.io> DanielePettenuzzo <daniele@px4.io>
+David Riseborough <drisebor@hotmail.com>
+David Sidrane <david_s5@nscdg.com>
+Dennis Mannhart <dennis@yuneecresearch.com> Dennis Mannhart <dennis.mannhart@gmail.com>
+Dennis Mannhart <dennis@yuneecresearch.com> Dennis Mannhart <dennis@px4.io>
+Elia Tarasov <elias.tarasov@gmail.com>
+Florian Achermann <florian.achermann@mavt.ethz.ch>
+Hamish Willee <hamishwillee@gmail.com>
+Ivo Drescher <ivo.drescher@gmail.com>
```
Comment (LorenzMeier) [inline .mailmap]: ETH Zurich

File: .mailmap [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+Jake Dahl <jacob.dahl@tealdrones.com>
+Johannes Brand <johannes@auterion.com>
+Joshua Whitehead <josh.whitehead@dornerworks.com>
+Julian Oes <julian@oes.ch> Julian Oes <joes@student.ethz.ch>
```
Comment (LorenzMeier) [inline .mailmap]: Yuneec Research

File: .mailmap [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+Julien Lecoeur <julien.lecoeur@gmail.com>
```
Comment (LorenzMeier) [inline .mailmap]: EPFL

File: .mailmap [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+Jun-Tao <jt.li@yuneec.com>
+Karl Schwabe <ksschwabe@gmail.com>
+Lorenz Meier <lorenz@px4.io> Lorenz Meier <lm@inf.ethz.ch>
```
Comment (LorenzMeier) [inline .mailmap]: Auterion

File: .mailmap [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+Matthias Grob <maetugr@gmail.com>
```
Comment (LorenzMeier) [inline .mailmap]: Yuneec Research

File: .mailmap [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+Phillip Kocmoud <phil@itchy.io>
+Roman Bapst <roman@px4.io> Roman Bapst <bapstroman@gmail.com>
+Roman Bapst <roman@px4.io> Roman Bapst <romanbapst@yahoo.de>
+Roman Bapst <roman@px4.io> Roman Bapst <bapstr@ethz.ch>
+Roman Bapst <roman@px4.io> tumbili <roman@px4.io>
+Roman Bapst <roman@px4.io> Roman <romanbapst@gmail.com>
+Roman Bapst <roman@px4.io> Roman <bapstroman@gmail.com>
+Roman Bapst <roman@px4.io> Roman <romanbapst@gmail.com>
+Roman Bapst <roman@px4.io> Roman <bapstr@ethz.ch>
+Roman Bapst <roman@px4.io> tumbili <bapstr@ethz.ch>
```
Comment (mrpollo) [inline .mailmap]: @RomanBapst I had fun chasing down your credentials over time, you have an identity problem!

**Comment by mrpollo** [comment]
Closing, this will be part of a new repo.

**Pull request closed without merging**